pkg/docs/data/cloudinit.tpl

```diff
@@ -1,10 +1,10 @@
 #cloud-config
 users:
-  - name: {{ or .Username "admin" }}
+  - name: {{ or .Username "myuser" }}
     sudo: ALL=(ALL) NOPASSWD:ALL
     ssh_authorized_keys:
       {{- range .AuthorizedKeys}}
       - {{.}}
       {{- else }}
       - ssh-rsa AAAA...
-      {{- end}}
```

Comment:
hmm may be cleaner to keep the newline here unless I miss something

Reply:
hmm, I think there was no newline in the file originally, and this PR added a new line. Do we want to keep the original version (without newline?)

Reply:
oh right my bad, definitely better to keep a newline